### PR TITLE
[Task] Refactor styles and contents

### PR DESCRIPTION
**Pull request by roseaugusto**
## Related Links

- Task Link: https://www.notion.so/Refactor-styles-and-contents-024a8d55d38a4b7ebd05cd2d947389dc
- Route Link: /

## Description

- Please check hardcoded text on the landing page and transfer to src/utilities/constants.js
- Replace double-quoted strings with single quotes.
- Utilize the use of main theme colors
  - 'main-backgound': '#F9F5F1' ----> bg-orange-100
  - 'footer-color' : '#594C4A'. -----> text-brown-900

## Notes

N/A

## Screenshots

N/A

### CODE REVIEW

**Review by charlotteds — COMMENTED** [review]
We still have some double-quoted strings in the footer section and constants file.
_Be careful with texts that contain single quotes._
image

**Comment by ume45876139** [comment]
# Comment
## Other pages
<details>
<summary>Screenshot</summary>

スクリーンショット 2022-04-29 16 51 57

</details>

- [ ] About other pages, text is overlapping with header
    - About
    - FAQs
    - Privacy Policy
    - Terms & Conditions

# Testing data
test device: Chrome
test time: 0.25h

File: src/pages/components/externalPages/ExternalPageTemplate.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import FooterContent from '../footerSection/FooterContent';
 const ExternalPageTemplate = ({ title, children }) => {
   return (
     <Fragment>
-      <header className='bg-main-backgound h-[100px] border-b border-brown-900 flex items-center sticky top-0'>
```
Comment (egiegarciano14) [inline src/pages/components/externalPages/ExternalPageTemplate.js:8]: Please don't remove the bg color, just change to bg-orange-100

Comment (roseaugusto) [reply src/pages/components/externalPages/ExternalPageTemplate.js:8]: I accidentally remove it sorry 😅

**Comment by ume45876139** [comment]
# Comment
## Footer
<details>
<summary>Screenshot</summary>

スクリーンショット 2022-04-29 17 27 09


</details>

- [ ] Text position is too higher 'Links'

# Testing data
test device: Chrome
test time: 0.25h

**Review by ume45876139 — APPROVED** [review]
# Working
## Chrome
<details>
<summary>PC</summary>


https://user-images.githubusercontent.com/39412730/165923715-4763c4a2-f59b-4076-bbd5-67f5a7228125.mov


</details>

<details>
<summary>iPad</summary>


https://user-images.githubusercontent.com/39412730/165923727-76a510df-467d-4382-b4e3-e41daa4cf834.mp4


</details>

<details>
<summary>iPhone</summary>

IMG_8025

</details>

- Links text position is fixed
- All functionalities are working

# Testing data
test device: Chrome, Safari
test time: 0.25h
comment: 2
iteration: 2
total test time: 0.75h

**Review by egiegarciano14 — APPROVED** [review]

**Pull request merged**